### DISCUSSION TITLE
Libical v3.0.15

### DISCUSSION THREAD

**Discussion by winterz** [libical/libical#595] [Announcements]
I'm planning to release Libical 3.0.15 soon, probably this weekend.
Any objections?

ReleaseNotes:
 * Add missing property parameters into libical-glib
 * Fix CMake option USE_32BIT_TIME_T actually uses a 32-bit time_t value
 * Fix icaltime_as_timet, which returned incorrect results for years >= 2100,
   to work properly between years 1902 and 10k.
 * Fix x-property comma handling and escaping

Also, for our build-in zoneinfo I may generate a VTIMEZONE for each time zone alias, like the Cyrus project does. I haven't fully investigated/tested this yet.

**Comment by ksmurchison** [comment]
I have no outstanding work other than the vCard stuff, so no objections.

**Comment by winterz** [comment]
my pre-release checklist is done.
will release 3.0.15 soon
